scanner/scanner.go

```diff
@@ -0,0 +1,40 @@
+package scanner
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	//"github.com/Layr-Labs/eigenda/api/grpc/scanner"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/Layr-Labs/eigensdk-go/metrics"
+)
+
+// Scanner struct
+type Scanner struct {
+	Config  *Config
+	Logger  logging.Logger
+	Metrics *Metrics
+}
+
+// NewScanner creates a new scanner with the provided config.
+func NewScanner(config *Config, logger logging.Logger) (*Scanner, error) {
+	promReg := prometheus.NewRegistry()
+	eigenMetrics := metrics.NewEigenMetrics(AppName, ":"+config.MetricsPort, promReg, logger.With("component", "EigenMetrics"))
```

Comment:
Does the scanner need eigen metrics? Doesn't seem relevant

Reply:
Yea, I wasn't sure so I added. Will remove